template.json

```diff
@@ -24,24 +31,13 @@
   },
   "Conditions": {
     "ShouldPartitionTraffic": {
-      "Fn::Not": [
-        {
-          "Fn::Equals": [
-            {
-              "Ref": "EIPAllocationIdBeta"
-            },
-            ""
-          ]
-        }
-      ]
+      "Fn::Not": [{"Fn::Equals": [{"Ref": "EIPAllocationIdBeta"}, ""]}]
```

Comment:
I think single line for this type of condition nesting is just so much more readable, can revert if there's no consensus (I know it's very likely against JSON best practices)